docs-new/docs/graphql/core/guides/integrations/aws-cognito.mdx

```diff
@@ -104,7 +104,7 @@ follow the steps below:
 3. After successfully logging in, you will be redirected to
 `https://localhost:3000/cognito-callback#id_token=xxxxxx&yyyyyy`. This
 page may be a 404 if you don't have a UI running on localhost:3000.
-Extract the `id_token` value from this URL.
+Extract the `id_token` value from this URL. {#test-cognito}
```

Comment:
The custom id syntax works only for headers

for content we have to rely on an empty div/span with the respective id

please try adding an empty div below this line. Please do similar changes in all other occurrences.

`<div id="test-cognito" />`